alarm/utils.py

```diff
@@ -20,13 +20,14 @@ def any_of_trade_pair_thresholds_is_broken(trade_pair):
     last_candle = Candle.last_for_trade_pair(trade_pair=trade_pair)
     penultimate_candle = Candle.penultimate_for_trade_pair(trade_pair=trade_pair)
 
-    for threshold in thresholds:
-        threshold_broken = threshold_is_broken(threshold.price, last_candle, penultimate_candle)
-        logger.info(f"{str(trade_pair).upper()}; "
-                    f"candles: {penultimate_candle}, {last_candle}; "
-                    f"threshold: {threshold}; "
-                    f"threshold broken: {threshold_broken}")
-        if threshold_broken:
-            return True
-
-    return False
+    if last_candle is not None and penultimate_candle is not None:
```

Comment:
if last_candle is None or penultimate_candle is None:
   return False
   
Flat is better than nested. Decrease a nesting level.

Reply:
++